pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -32,10 +32,19 @@ func (n *NetworkSpec) IsMultus() bool {
 	return n.Provider == NetworkProviderMultus
 }
 
-// IsHost get whether to use host network provider. This method also preserve
-// compatibility with the old HostNetwork field.
+// IsHost can be used to determine if  pods should use host networking by
+// checking whether  the cephCluster is configured to use the "host" network provider.
+// This method  also maintains compatibility with the old hostNetwork setting
```

Comment:
Note to check space usage in doc comments. I see several double-spaces here and below. I'm not too concerned about this in draft status, but it might be good to check if the editor you're using makes it harder to see these. I have whitespace markers turned on in mine, and I find it very helpful for visualizing space and tab usage.

```suggestion
// IsHost can be used to determine if pods should use host networking by
// checking whether the cephCluster is configured to use the "host" network provider.
// This method also maintains compatibility with the old hostNetwork setting
```

Reply:
done